pkg/controller/upload_controller.go

```diff
@@ -172,10 +186,22 @@ func (c *uploadController) processUploadItem(key string) error {
 		// For UploadPhaseInProgress, the resource lease logic will process the Upload if the lease is not held by
 		// another DataManager. If the DataManager holding the lease has died and/or lease has expired the current node
 		// will pick such record in UploadPhaseInProgress status for processing.
+	case pluginv1api.UploadPhaseCanceled:
```

Comment:
Same as before. Move up?

Reply:
golang doesn't have fallthrough for switch statement.